src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -557,8 +562,8 @@ To provide a new name, use the same pragma and provide a name as parameter:
 If the job's name is not set, the ProActive kernel uses the current notebook name, if possible, or gives a random one.
 
 ==== List all submitted jobs
-
-To get all submitted job ids and names, use `list_submitted_jobs` pragma this way:
+s
```

Comment:
s?

Reply:
I really don't know from where it came. xD